test/Plutip/Contract.purs

```diff
@@ -941,7 +942,7 @@ suite = do
 
           ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
```

Comment:
I noticed than now the API is quite inconsistent. Please make the same change to `mkUnbalancedTx`: rename the non-throwing variant to `E`, the short name should be throwing.